src/shared/parse-helpers.c

```diff
@@ -14,6 +14,12 @@ int parse_arg_signal(const char *s, int *ret) {
                 return 0;
         }
 
+        if (streq(s, "list")) {
+                for (int i = 1; i < _NSIG; i++)
+                        printf("%d\t%s\n", i, signal_to_string(i));
```

Comment:
i'd include "SIG" in the table... i.e. SIGTERM is bettern known than just "TERM"...

Also, I'd generate the table with format-table.c, so that the left column can be aligned to the right